### PR TITLE
finished adding half-orc

### PR DESCRIPTION
**Pull request by Texdom65**
added script and code

### CODE REVIEW

File: character/database/racial_adjustment.py [hunk 1]
```diff
@@ -22,6 +22,12 @@ def get_racial_adjust(character):
         character.strength = character.strength - 1
         return character
 
+    elif character.race == "half-orc":
+        character.strength = character.strength + 1
```
Comment (JoshuaKunz) [inline character/database/racial_adjustment.py:26]: These can be simplified using += 😎

Comment (Texdom65) [reply character/database/racial_adjustment.py:26]: 😋

**Review by JoshuaKunz — CHANGES_REQUESTED** [review]
Simplify code to use += to maximize readability

**Review by JoshuaKunz — APPROVED** [review]

**Comment by JoshuaKunz** [comment]
@Texdom65 this can be merged

**Pull request merged**